cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -95,17 +99,28 @@ type indexArgs struct {
 
 	// ShardMerging is true if we want zoekt-git-index to respect compound shards.
 	ShardMerging bool
+
+	// UseSourcegraphIDForName is true if we want to use the Sourcegraph ID as prefix for the shard name.
+	UseSourcegraphIDForName bool
 }
 
 // BuildOptions returns a build.Options represented by indexArgs. Note: it
 // doesn't set fields like repository/branch.
 func (o *indexArgs) BuildOptions() *build.Options {
+
+	// Default to tenant 1 if no tenant is set.
+	tenantID := o.TenantID
+	if o.TenantID < 1 {
+		tenantID = 1
+	}
```

Comment:
Maybe mark this code as temporary. This is the sort of thing we want to exist only during rollout? Or maybe even give up if not set instead?